modules/tracking/include/opencv2/tracking/kalman_filters.hpp

```diff
@@ -0,0 +1,215 @@
+#pragma once
+#include "opencv2/core.hpp"
+#include <limits>
+
+namespace cv
+{
```

Comment:
put the functionality into nested namespace, `namespace cv { namespace tracking { ... } }`